pretest.js

```diff
@@ -26,7 +26,7 @@ if (!process.env.MSSQL_DATABASE) process.env.MSSQL_DATABASE = global.getConfig()
 const catFileCmd = (isWin ? 'type ' : 'cat ') + sqlFileDir;
 
 const sqlcmd = catFileCmd + ' | ' + sqlDependencyDir + ' -s ' + process.env.MSSQL_HOST + ' -o ' + process.env.MSSQL_PORT +
-' -u ' + process.env.MSSQL_USER + ' -p ' + process.env.MSSQL_PASSWORD + ' -d ' + process.env.MSSQL_DATABASE;
+' -u ' + process.env.MSSQL_USER + ' -p "' + process.env.MSSQL_PASSWORD + '" -d ' + process.env.MSSQL_DATABASE;
```

Comment:
Does it make sense to quote all vars?

Reply:
My database password is "yourStrong (!) Password" and without quotes, it doesn't work

Reply:
I understand. I meant to say if we should use `"` for user and database names too.

Reply:
Sorry, I had not understood! Resolved